lib/deref.js

```diff
@@ -309,8 +295,7 @@ module.exports = {
       }
       else {
         return {
-          type: 'string',
-          default: 'schema type not provided'
+          type: 'object'
```

Comment:
Most of resolvers treats default type as `object` and not a `string`. Although `type` is somewhat required property, I have seen schema where schema contains only `title` and `description` and empty `object` would be more appropriate than `string` in such cases.